src/content/teams/14_xen.mdx

```diff
@@ -2,18 +2,6 @@
 name: Xen Project Hypervisor Team
 description: Maintains the Xen Project Hypervisor tooling and the accompanying NixOS modules.
 members:
-  - name: Fernando Rodrigues
```

Comment:
Is it on purpose to remove members from the Xen team?

Reply:
Just put this together at FOSDEM, probably deleted this by accident in the rush. Just added it back :D